packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputShadowNode.java

```diff
@@ -53,10 +53,7 @@ public class ReactTextInputShadowNode extends ReactBaseTextShadowNode
   public ReactTextInputShadowNode(
       @Nullable ReactTextViewManagerCallback reactTextViewManagerCallback) {
     super(reactTextViewManagerCallback);
-    mTextBreakStrategy =
-        (Build.VERSION.SDK_INT < Build.VERSION_CODES.M)
-            ? Layout.BREAK_STRATEGY_SIMPLE
-            : Layout.BREAK_STRATEGY_HIGH_QUALITY;
+    mTextBreakStrategy = Layout.BREAK_STRATEGY_SIMPLE;
```

Comment:
mmm, tho i understand why you are making this change, i'm concerned that this will break apps that might be assuming HQ.

is there a reason we want to change this at the infra level instead of at the product layer besides inconsistency with website docs?

Reply:
Nope. Was just trying to fix the react-native behaviour as per the documentation. Typing in a TextInput with highQuality break strategy is super annoying as the text wrapping keeps on changing with user input.